os/binfmt/binfmt_exit.c

```diff
@@ -129,13 +129,19 @@ int binfmt_exit(FAR struct binary_s *bin)
 		}
 		address = (FAR void *)sq_next((FAR sq_entry_t *)address);
 	}
-
+#ifdef CONFIG_OPTIMIZE_APP_RELOAD
+	if (!bin->reload)
+#endif
 	/* Free the RAM partition into which this app was loaded */
-	mm_free_ram_partition((uint32_t)bin->uheap);
+	mm_free_ram_partition((uint32_t)bin->ramstart);
```

Comment:
Please use the same policy for this case.
In binfmt_loadbinary.c, line 112, you give different indentation.
```
#ifdef AA
  if (xx)
#endif 
  yyyyy;

or

#ifdef AA
  if (xx)
#endif 
     yyyyy;
```

Reply:
Added indentation.